k8s/kontaind/deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
               mountPath: /var/lib/kubelet/device-plugins
       initContainers:
         - name: kontain-installer
-          image: docker.io/kontainapp/kontain-installer:latest
+          image: docker.io/kontainapp/runenv-kontain-installer:latest
```

Comment:
I did not know the images is runenv-* - why ? does it have KM run environment ?

Reply:
Because you asked me to use image.mk to create the image. The naming is controlled by image.mk.

Reply:
ah, got it.